src/umpire/op/CudaAdviseOperation.hpp

```diff
@@ -7,13 +7,17 @@
 #ifndef UMPIRE_CudaAdviseAccessedByOperation_HPP
```

Comment:
To be consistent with ending comment on line 37, I suggest

```suggestion
#ifndef UMPIRE_CudaAdviseOperation_HPP
```